### PR TITLE
Task/display relative time update search

### PR DESCRIPTION
**Pull request by jennywin**
@cchanningallen 
1. All created_at attributes now display as relative time ago (e.g., "yesterday at 2:58pm).
2. Updated search to include newly added models + updated search results view. Also added the PostgreSQL trigram package to allow for matching of any three-character subsets.

### CODE REVIEW

File: app/views/articles/show.html.erb [hunk 1]
```diff
@@ -10,12 +10,17 @@
     <div class="panel panel-default">
       <div class="panel-heading">
         <h2 class="panel-title-1"><%= @article.name %></h2>
-        <h4>Author: <%= @article.user.full_name %></h4>
+        
+        <h4>Author</h4>
+        <p><%= @article.user.full_name %></p>
+        
+        <h4>Created</h4>
+        <p><%= local_time_ago(@article.created_at) %></p>
```
Comment (cchanningallen) [inline app/views/articles/show.html.erb:18]: :fire: :fire:

File: app/views/searches/index.html.erb [hunk 2]
```diff
@@ -3,8 +3,25 @@
...
+    <p><%= link_to pgsd.searchable.article.name %>: <%= pgsd.searchable.content %></p><hr>
+  <% elsif pgsd.searchable.class == Reference %>
+    <h3>Resources</h3>
+    <p><%= link_to pgsd.searchable.name, pgsd.searchable %></p><hr>
+  <% elsif pgsd.searchable.class == Category %>
+    <h3>Categories</h3>
+    <p><%= link_to pgsd.searchable.name, pgsd.searchable %></p><hr>
+  <% elsif pgsd.searchable.class == Tag %>
+    <h3>Tags</h3>
+    <p><%= link_to pgsd.searchable.name, pgsd.searchable %></p><hr>
```
Comment (cchanningallen) [inline app/views/searches/index.html.erb:25]: Rad! ALL THE THINS :godmode:

**Comment by cchanningallen** [comment]
This looks awesome! Good job :)

+1

**Comment by jennywin** [comment]
@cchanningallen Thank you!

**Pull request merged**